os/arch/arm/src/armv8-m/mpu.h

```diff
@@ -687,6 +687,29 @@ static inline void up_set_mpu_app_configuration(struct tcb_s *rtcb)
 #define up_set_mpu_app_configuration(x)
 #endif
 
+#if defined(CONFIG_MPU_STACKGUARD)
+static inline void up_set_mpu_region_configuration(struct tcb_s *rtcb, int region_no)
```

Comment:
If we are going to use this API only for stack protection, then we can rename it as up_set_mpu_stack_gaurd